realm/realm-library/src/main/java/io/realm/internal/ImplicitTransaction.java

```diff
@@ -16,6 +16,8 @@
 
 package io.realm.internal;
 
+import io.realm.internal.async.BadVersionException;
```

Comment:
"BadVersion" just sounds "bad" - and doesn't say much. (I know it comes from core). Can't we find a better name that says more?